README.md

```diff
@@ -7,6 +7,8 @@ Most of these Extensions can be downloaded on the respective GitHub pages, usual
 If you wish to add your Extension to this list, create a PR!
 
 ### Currently working
+- **Jukebox Extended Reborn**: https://github.com/spartacus04/jext-reborn <br>
```

Comment:
The list is currently in alphabetical order, please add your extension in the correct spot :)